canonical/ubuntu.com
# feat: Add ubuntu summit 24 page

## Done

- [x] Added website ubuntu summit 2024.

## QA

- Go to the below demo link and check if there are no regressions, check responsiveness on different devices (mobile, tablet, etc.)
- https://ubuntu-com-14316.demos.haus/summit

## Issue / Card

Fixes [WD-15079](https://warthogs.atlassian.net/browse/WD-15079)

## Screenshots

![](https://github.com/user-attachments/assets/d8708ba8-936a-4c2d-b32b-4aebf07830c9)



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-15079]: https://warthogs.atlassian.net/browse/WD-15079?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
